src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -1881,20 +1881,22 @@ void FlexPA::genInstRowPatternInit(std::vector<FlexDPNode>& nodes,
 void FlexPA::genInstRowPatternPerform(std::vector<FlexDPNode>& nodes,
                                       const std::vector<frInst*>& insts)
 {
-  for (int curr_idx_1 = 0; curr_idx_1 <= (int) insts.size(); curr_idx_1++) {
-    for (int curr_idx_2 = 0; curr_idx_2 < ACCESS_PATTERN_END_ITERATION_NUM;
-         curr_idx_2++) {
+  for (int curr_pin_idx = 0; curr_pin_idx <= (int) insts.size();
```

Comment:
why not curr _inst_idx?

Reply:
Oversight. I actually have already caught that on a further PR I'm working on, but did not think to fix it here. Nice catch. I'm fixing this an re requesting review.